ledger/participant-integration-api/src/main/scala/platform/apiserver/services/LedgerConfigProvider.scala

```diff
@@ -59,10 +59,16 @@ private[apiserver] final class LedgerConfigProvider private (
   // - Mark the provider as ready if no configuration was found after a timeout
   // - Submit the initial config if none is found after a delay
   startLoading()
-  materializer.scheduleOnce(config.configurationLoadTimeout.toNanos.nanos, () => {
-    readyPromise.trySuccess(())
-    ()
-  })
+  materializer.scheduleOnce(
+    config.configurationLoadTimeout.toNanos.nanos,
+    () => {
+      if (readyPromise.trySuccess(())) {
+        logger.warn(
+          s"No ledger configuration found after ${config.configurationLoadTimeout}. The ledger API server will now be ready but all services that depend on the ledger configuration will return UNAVAILABLE.")
```

Comment:
```suggestion
          s"No ledger configuration found after ${config.configurationLoadTimeout}. The ledger API server is now responding to requests, but all services that depend on the ledger configuration will return UNAVAILABLE.")
```

Reply:
_Technically_ it's not responding to requests NOW, but in a very short time in the future - there is some code running between `LedgerConfigProvider.ready` and the binding of the network port in `GrpcServer.Owner.acquire`. How about the following?
```
The ledger API server will now start but all services that depend on the ledger configuration will return UNAVAILABLE until at least one ledger configuration is found.
```